ktor-http/ktor-http-cio/common/src/io/ktor/http/cio/Pipeline.kt

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
May be it's better to leave `Pipeline`, `ServerPipeline`, `ServerIncomingConnection` and `ServerRequestScope` files in package `io.ktor.server.cio.backend` and move them into `ktor-server-cio` module, which already contains this package?

Reply:
@e5l WDYT about it? it's better to do it just now in 2.0, where there are much changes in server packages

Reply:
any news on this?

Reply:
Will do it in the separate PR. This one is already huge.